.circleci/config.yml

```diff
@@ -21,6 +21,16 @@ jobs:
           name: Publish layer
           command: make publish-nodejs14x-ci
 
+  publish-nodejs16x:
+    docker:
+      - image: circleci/python:3.8
```

Comment:
is this the right image

Reply:
I was wondering about that as well, but the other Node layers use the CI Python images. The actual publishing script is a Bash job. The Node assets are contents for the layer, and aren't actually run during the build.